netcontrold/lib/dataif.py

```diff
@@ -184,7 +184,7 @@ def add_rxq(self, _id):
         # check if this rxq is already available.
         rxq = self.find_rxq_by_id(_id)
         if rxq:
-            return rxq
+            raise ObjConsistencyExc("rxq %d is available" % _id)
```

Comment:
"modified add_rxq function in dataif.py file" in commit subject!.. What did you modify and for what ?.. I nack the commit for proper description of patch.